src/main/resources/xml/schema/2020-1/nordic2020-1.nav-references.sch

```diff
@@ -125,9 +125,14 @@
             <let name="document-in-nav" value="((preceding::html:a | self::*) intersect ancestor::html:nav//html:a)[substring-before(@href,'#') = $href][1]"/>
             <let name="document-in-nav-depth" value="count($document-in-nav/ancestor::html:li)"/>
             <let name="depth-in-nav" value="count(ancestor::html:li)"/>
-            <let name="depth-in-content" value="$result-ref-first/xs:integer(@data-heading-depth)"/>
 
-            <assert test="not(@data-heading-depth) or not($result-ref-first) or $depth-in-nav = $depth-in-content">[nordic_nav_references_2b] The nesting of headlines in the content does not match the
+            <let name="data-part-depth"
+                 value="if ($result-ref-first/preceding-sibling::c:result[@data-heading-element='h1'][1]/
+                  tokenize(@data-document-epub-type, '\s+') = 'part' or @data-document-role = 'doc-part') then 1 else 0" />
```

Comment:
This will work in most cases. But if a part has a `aria-label` heading, then it won't work. I think we need to handle those cases as well.